gui/src/components/s3/account-management.vue

```diff
@@ -836,4 +853,4 @@ tbody tr:active {
 .credentials-item-label {
   width: 10rem;
 }
-</style>
+</style>
```

Comment:
Every file should have a new line at the end of the file, please re add that.

Reply:
code formatted with prettier